src/DotNet.Testcontainers.Tests/Unit/Containers/Unix/Database/OracleTestcontainerTest.cs

```diff
@@ -21,5 +23,18 @@ public void CanQueryContainerUsingProvidedConnectionString()
       // Assert
       Assert.NotNull(connectionString);
     }
+
+    [Fact]
+    public async Task CanConnectToOracleContainerAndOpenConnection()
+    {
+      // Arrange
+      await using var connection = new OracleConnection(this.oracleFixture.OracleTestcontainer.ConnectionString);
+
+      // Act
+      await connection.OpenAsync();
```

Comment:
You need to start the container first.

Reply:
https://mderriey.com/2017/09/04/async-lifetime-with-xunit/
InitializeAsync will be called right after the constructor is invoked, and DisposeAsync just before Dispose - if it exists - is called.

Reply:
I did not know the integration in xUnit, thanks 👍!

- I guess we can remove `IDisposable` and `Dispose` in `OracleFixture` then.
- `InitializeAsync` and `DisposeAsync` can return the Task immediately.
- `DisposeAsync` won't need `StopAsync`.

How should we continue with exports? Expect that everything is working?

Reply:
I make the appropriate changes to the latest comments in last commit, yes i expected that everything is working

Reply:
Tell me in more detail what you mean

> How should we continue with exports

Reply:
> Tell me in more detail what you mean

If we use the exports (`export ORACLE...`, etc.) in the WaitStrategy, we cannot use other Docker images based on Oracle, because the paths may be different.

Reply:
Now on the docker hub there is only one official version from Oracle and this is 12, but many projects still use version 11. Since my working project also uses version 11, I found a custom image of version 11 of Oracle and applied it in the current implementation, I I think that the absolute path for him will not change.

Reply:
I hope I correctly understood your remark

Reply:
> Now on the docker hub there is only one official version from Oracle and this is 12, but many projects still use version 11 [...].

Ok, let's fix the test (`CanConnectToOracleContainerAndOpenConnection`) and get the PR ready to merge. Please consider to export the variables (if necessary) like we already did with the MySql container e. g.: https://github.com/HofmeisterAn/dotnet-testcontainers/blob/0a3930ddff2eb5d3cf0e2dc433301adcef3990a6/src/DotNet.Testcontainers/Containers/Configurations/Databases/MySqlTestcontainerConfiguration.cs#L10

If you need help fixing the database connection test let me know, I'm happy to help.

Reply:
Ok, i fix waitStrategy now we don't exporting the variables